src/Propulsion.Cosmos/PropulsionCosmosPrometheus.fs

```diff
@@ -4,111 +4,155 @@ namespace Propulsion.CosmosStore.Prometheus
 namespace Propulsion.Cosmos.Prometheus
 #endif
 
+(*
```

Comment:
Can you convert this to a ~2 line `///` comment at the top to fit house style please? (but definitely thanks for adding it)

Reply:
Done